### PR TITLE
Marriage abroad/flatten cyprus

### PR DESCRIPTION
**Pull request by cbaines**
Trello: https://trello.com/c/OPt0bqUy/624-%283%29-marriage-abroad-improvement%3A-cyprus-%282-outcomes%29

This is dependent on https://github.com/alphagov/smart-answers/pull/3062 currently, as #3062 solves some problems with carriage returns in the test artefacts and erb templates.

### CODE REVIEW

File: lib/smart_answer_flows/marriage-abroad/outcomes/countries/cyprus/ceremony_country/partner_british/_same_sex.erb [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+
+Once you’ve submitted these and paid the registration fee (see below), the High Commission will display your notice publicly for 14 days.
+
+As long as nobody has registered an objection after this time, the registration officer can then register your partnership any time until 3 months after the date you gave notice.
+
+You’ll need to provide two witnesses and pay a fee to register your civil partnership. You’ll need to pay an additional fee for your civil partnership certificate (see below).
+
+## Fees
+
+Service | Fee
```
Comment (chao-xian) [inline lib/smart_answer_flows/marriage-abroad/outcomes/countries/cyprus/ceremony_country/partner_british/_same_sex.erb]: We should still be using the fees partial: `lib/smart_answer_flows/marriage-abroad/outcomes/_consular_fees_table_items.govspeak.erb` iirc.

Comment (chao-xian) [inline lib/smart_answer_flows/marriage-abroad/outcomes/countries/cyprus/ceremony_country/partner_british/_same_sex.erb]: See https://github.com/alphagov/smart-answers/blob/1628ecdaa2aa06698682653676c6903e38c49cf0/lib/smart_answer_flows/marriage-abroad/outcomes/countries/colombia/ceremony_country/partner_british/_opposite_sex.govspeak.erb#L16

File: lib/smart_answer_flows/marriage-abroad/outcomes/countries/cyprus/ceremony_country/partner_british/_same_sex.erb [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+-|-
+Receiving a notice of registration | £50
+Registering a civil partnership | £150
+Issuing a civil partnership certificate | £50
+
+You normally have to pay fees for consular services in the local currency - these are shown in the list of [consular fees for Cyprus](/government/publications/cyprus-consular-fees).
+
+You can pay by cash or credit card, but not by personal cheque.
```
Comment (chao-xian) [inline lib/smart_answer_flows/marriage-abroad/outcomes/countries/cyprus/ceremony_country/partner_british/_same_sex.erb]: Similarly https://github.com/alphagov/smart-answers/blob/1628ecdaa2aa06698682653676c6903e38c49cf0/lib/smart_answer_flows/marriage-abroad/outcomes/countries/colombia/ceremony_country/partner_british/_opposite_sex.govspeak.erb#L23

**Comment by cbaines** [comment]
Thanks for following up on this in my absence everyone :+1:

**Pull request merged**